### PR TITLE
Add script for vending testtrack-cli

### PR DESCRIPTION
**Pull request by aburgel**
/domain @jmileham @smudge @samandmoore 
/no-platform

Added a step to the `vendor_deps` task to download the CLI to the correct directory (we had multiple versions committed in different directories).

Also, upgraded the CLI to v0.9.7.

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -32,6 +32,21 @@ task :vendor_deps do
   sh 'bower install'
   sh 'cp', 'bower_components/test_track_js_client/dist/testTrack.bundle.min.js', 'app/assets/javascripts'
 
+  # Download testtrack-cli
+  TEST_TRACK_CLI_VERSION = 'v0.9.7'
```
Comment (smudge) [inline Rakefile]: So if we want to upgrade the vendored CLI version, do we just change this constant?

Comment (aburgel) [reply Rakefile]: yep, and then run `bundle exec rake vendor_deps`... there's a bit of noise w/ fakeable_her.  we are not running the latest for reasons that i don't understand (you actually addressed it in #65). so you need to not commit those changes.

Comment (jmileham) [reply Rakefile]: should we rip out the rest of the gem upgrades and let them rot?

File: Rakefile [hunk 2]
```diff
@@ -32,6 +32,21 @@ task :vendor_deps do
...
+  FileUtils.module_eval do
+    mkdir_p 'vendor/bin/testtrack-cli'
+    cd 'vendor/bin/testtrack-cli' do
+      rm_r Dir.glob('*')
+
+      %w(darwin linux).each do |arch|
+        `curl -L https://github.com/Betterment/testtrack-cli/releases/download/#{TEST_TRACK_CLI_VERSION}/testtrack.#{arch} \
```
Comment (jmileham) [inline Rakefile]: One fun thing I learned is that this gets rate limited pretty readily when used unauthenticated. I spose it doesn't matter much for this operation, though, you only rarely need to do this, and can cope with some pain if it arises.

**Comment by jmileham** [comment]
<< domain lgtm

File: Rakefile (hunk 1 above)
Comment (aburgel) [reply Rakefile]: works for me, we haven't touched those gems in a long time. i can do this separately.

**Pull request merged**